addons/folo/jaxrs/src/main/java/org/commonjava/indy/folo/bind/jaxrs/TrackingAdminResource.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (C) 2023 Red Hat, Inc. (https://github.com/Commonjava/indy)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.indy.folo.bind.jaxrs;
```

Comment:
As folo addon will be dropped in future, I don't recommend to set this in folo package. Maybe you should add it in content package, like org.commonjava.indy.core.bind.jaxrs in bindings module. (So means the related model class should be also moved to other parts like core)

Reply:
moved according to your suggestion